etc/examples/paths.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# these can be set by folks running the examples/Makefile.testing to use a different binaries than in the path
+ECHO=${ECHO:-echo}
+JQ=${JQ:-jq}
+KUBECTL=${KUBECTL:-kubectl}
```

Comment:
I don't see any version-sensitive uses of kubectl that would require this.

Reply:
This is to allow *different versions* of kubectl to be easily used by testing routines down the road, same as the specification of pachctl.